### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by HENRYKC24**
### In this pull request, I did these:point_down:
1. I deleted all the boilerplate from CRA (text, images, styles).
2. I created a directory called components.
3. Inside components I created a new Calculator.js file.
4. In Calculator.js, I created a React component that matches this design in the requirement.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```
Comment (Dipeshtwis) [inline src/App.js:27]: - [ ] I would not recommend removing the `App` component. Use the `App.js` as the parent component and it should have the imported `Calculator.js` component.

- [ ] Also, The `App.js` component should be class-based. In this milestone, the objective is to practice the class-based React component.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    this.state = {};
+  }
+
+  render() {
+    const tableData = [
+      [0],
+      ['AC', '+/-', '%', '+'],
+      [7, 8, 9, `${String.fromCharCode(215)}`],
+      [4, 5, 6, '-'],
+      [1, 2, 3, '+'],
```
Comment (Dipeshtwis) [inline src/components/Calculator.js]: - [ ] There are duplicate `+` signs there, instead, it should be the `÷` (division) sign in one of the place.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @HENRYKC24, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Well done with deleting all the boilerplate created by the CRA.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (HENRYKC24) [reply src/components/Calculator.js]: Thank you. I was wondering why we have two plus sign:smile:
My bad eye! I greatly appreciate:pray:

File: src/App.js (hunk 1 above)
Comment (HENRYKC24) [reply src/App.js:27]: I really removed this file. I am looking for it right now but still, it is not in my project. I think I deleted that file. Could you check if the commit is the most recent, please :pray:
Probably that is why the color is red, I guess.

Thank you for pointing that out, anyways. I learned a lot from this.:+1:
[Check this](https://screenrec.com/share/SIRgV0H9ZT)

Comment (Dipeshtwis) [reply src/App.js:27]: Yes, you have removed the file. Please add again and do the necessary changes. :smile:

Comment (HENRYKC24) [reply src/App.js:27]: Done!:+1:
Pull request resubmitted, also.

**Comment by Davidosky007** [comment]
## Hi @HENRYKC24 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
done-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by HENRYKC24** [comment]
Thank you so much @Davidosky007 :pray:

**Pull request merged**